ui/Modal/ManageRemotes.svelte

```diff
@@ -0,0 +1,93 @@
+<script lang="ts">
+  import { Variant as IllustrationVariant } from "../src/illustration";
+
+  import {
+    Illustration,
+    List,
+    RemoteListItem,
+  } from "../DesignSystem/Component";
+  import { Button, Input } from "../DesignSystem/Primitive";
+  import { Variant } from "../src/illustration";
+
+  const remotes = [
+    {
+      handle: "cloudhead",
+      peerID: "hwd1yreg4khbjfa4gsyrio3f7ehluwkdhyregs4k",
+      maintainer: true,
+      synced: true,
```

Comment:
What's the meaning of being synced with a peer? Couldn't imagine how we determine that at the moment.

Reply:
The idea behind it is that we have tracked them but haven't got their changes in our repo yet. So it's determined by having them tracked for that project, but no remote exists for them under that project.

Reply:
Might be a matter of wording then. We syncing as a mechanism in another way already.

Reply:
> but maybe is it too early to introduce the TS stuff? I'm not sure what the dev flow of UI components is :)

The component is already TS so no harm in modelling the data.

Reply:
> Might be a matter of wording then. We syncing as a mechanism in another way already.

Ya maybe something better is `type RemoteStatus = Tracking | Searching`. And actually, to refine the type above, it would be better represented by:
```ts
interface Tracked {
  kind: "tracked";
  handle: string;
  peerId: string;
  maintainer: boolean;
}

interface Searching {
  kind: "searching";
  peerId: string;
}

type RemoteStatus = Tracked | Searching;
```

Since we know nothing of the `Searching` variant other than the `peerId`, which gets rid of these **nasty** `null`s